src/plugins/intel_gna/frontend/model_quantizer.hpp

```diff
@@ -22,125 +22,125 @@
 namespace GNAPluginNS {
 
 /**
- * Quantize entire cnn - network
- * @tparam T - type trait for weights and biases
+ * Quantize entire network
  */
-template<class T>
 class ModelQuantizer {
  public:
-    InferenceEngine::CNNNetwork quantize(const InferenceEngine::CNNNetwork &model, float scaleFactor) const {
-        return quantize(model, [](const InferenceEngine::CNNNetwork &, bool runBeforeCopy, bool lowPrecision){}, scaleFactor);
+    // Below four functions are used only in unit tests
```

Comment:
That means that we need to remove them from the class, and define in the test unit class only.

Reply:
Can we agree to have it done in the next PR for mixed precision?